framework/db/target_manager.py

```diff
@@ -163,11 +175,9 @@ def AddTarget(self, TargetURL, session_id=None):
             target_id = config_obj.id
             self.CreateMissingDBsForURL(TargetURL)
             self.SetTarget(target_id)
+
         else:
-            session_obj = self.db.session.query(
```

Comment:
By removing these lines above in a ``if`` statement introduces a bug. If the target URL is already know but the user does not use ``-f``, then OWTF falls in the ``else`` statement where ``target_obj`` does not exist, hence an error ``UnboundLocalError``.